loading.bs

```diff
@@ -706,6 +706,8 @@ reporting to a [=signed exchange report=] |report|, returns a
     :: "`none`"
     : [=request/mode=]
     :: "`cors`"
+    : [=request/credentials mode=]
+    :: "`omit`"
```

Comment:
Pardon my naivete wrt the fetch monkeypatch,

https://fetch.spec.whatwg.org/#concept-request-credentials-mode says:
> Unless stated otherwise, it is "`omit`".

Is it stated otherwise somewhere? Or is duplicate to avoid confusion?

Reply:
Argh, thanks for pointing that out. Yep, this PR is a no-op. Never mind!

Reply:
Thanks for taking a look anyway. I thought this value default was changed, https://github.com/whatwg/fetch/pull/585. 

It might still be worthwhile to ensure it's set to "omit".

Reply:
It seems like "same-origin" would address the threat mentioned in the PR description ("another origin"); is there another threat you had in mind?

Reply:
Making the value `omit` would only be for completeness sake – `same-origin` gives the same level of protection but unnecessarily sends cookies for same origin cert requests.